test/performance/odp_dma_perf_run.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/sh
+#
+# Copyright (c) 2022, Nokia
+# All rights reserved.
+#
+# SPDX-License-Identifier:	BSD-3-Clause
+
+TEST_DIR="${TEST_DIR:-$(dirname $0)}"
+BIN_NAME=odp_dma_perf
+ROUNDS=1000000
+
+check_result()
+{
+	if [ $1 -ne 0 ]; then
+		echo "Test FAILED, exiting"
+		exit 1
+	fi
+}
+
+echo "odp_dma_perf: synchronous transfer 1"
+echo "===================================="
+
+${TEST_DIR}/${BIN_NAME}${EXEEXT} -t 0 -g 0 -i 6 -s 10240 -T 0 -r $ROUNDS
```

Comment:
OK, so have to always guess the characteristics of the least capable implementation instead of tailoring these performance scripts to the repo's implementation.

I'll change `odp_dma_perf` so that it will exit with 0 if attempted scenario is not supported, so unsupported tests are just skipped.

Reply:
Sounds like an OK solution. The application could also return 77 (exit code expected by automake for skipped tests). This way the output of `make check` will be more informative.
E.g.
```
PASS: odp_atomic_perf
PASS: odp_bench_packet
PASS: odp_crypto
SKIP: odp_dma_perf
PASS: odp_ipsec
```

Reply:
Fixed in v2 with support of #1475.